lm_eval/api/task.py

```diff
@@ -41,6 +41,22 @@
 eval_logger = logging.getLogger("lm-eval")
 
 
+@dataclass
+class GroupConfig(dict):
```

Comment:
Thoughts on the placement of `group_alias`, `task_alias` into GroupConfig and removing those attributes from `TaskConfig`?

Reply:
`TaskConfig` would still need it, `task_alias` as an alias for it's own task. And `group_alias` to accommodate how each task can be attributed to a group. I think I need to remove `GroupConfig` since I didn't end up using it.